superdesk/publish/subscribers.py

```diff
@@ -70,6 +72,13 @@ class SubscribersResource(Resource):
                 },
             },
         },
+        "schedule": {
+            "type": "dict",
+            "schema": {
+                "startDate": {"type": "string", "nullable": True},
+                "endDate": {"type": "string", "nullable": True},
```

Comment:
we should use snake case to keep consistent.
```suggestion
                "start_date": {"type": "string", "nullable": True},
                "end_date": {"type": "string", "nullable": True},
```